handler/src/test/java/io/netty/handler/pcap/PcapWriteHandlerTest.java

```diff
@@ -487,4 +521,161 @@ private static void verifyUdpCapture(boolean verifyGlobalHeaders, ByteBuf byteBu
         assertTrue(ipv4Packet.release());
         assertTrue(udpPacket.release());
     }
+
+    private static class EmbeddedDatagramChannel extends EmbeddedChannel implements DatagramChannel {
+        private final InetSocketAddress local;
+        private final InetSocketAddress remote;
+        private DatagramChannelConfig config;
+
+        EmbeddedDatagramChannel(InetSocketAddress local, InetSocketAddress remote) {
+            super(new CustomChannelId(1), false);
```

Comment:
Just use `DefaultChannelId` so we don't need to change the visibility of `CustomChannelId`